source/security/index.html.md.erb

```diff
@@ -31,40 +41,70 @@ To further secure your live developer keys:
-- reporting your PCI DSS compliance status for each of these unique MIDs to your acquiring bank
+* agreeing with your acquiring bank to allocate an unique MID for each
+separate payment channel you have in place
+* reporting your PCI DSS compliance status for each of these unique MIDs to
+your acquiring bank
 
-If you have one MID that encompasses multiple payment channels, the compliance requirements will be more complex for that MID. Check the PCI DSS for more information.
+If you have one MID that encompasses multiple payment channels, the compliance
+requirements will be more complex for that MID. Check the PCI DSS for more
+information.
 
 ### Determine your PCI DSS compliance requirements
 
-Your requirements depend on the number of transactions that you process as a merchant per scheme (Visa, MasterCard) per year. For example, if you process 4 million transactions with Visa and 3 million with MasterCard, the “Fewer than 6 million transactions per year” category still applies despite the fact that the total number of transactions is larger than 6 million. Your merchant level should be confirmed with your acquiring bank.
+Your requirements depend on the number of transactions that you process as a
+merchant per scheme (Visa, MasterCard) per year. For example, if you process 4
+million transactions with Visa and 3 million with MasterCard, the “Fewer than
+6 million transactions per year” category still applies despite the fact that
+the total number of transactions is larger than 6 million. Your merchant level
+should be confirmed with your acquiring bank.
 
 ### Assessing your compliance requirements
 
 #### Process fewer than 6 million transactions per year
 
-If you process fewer than 6 million transactions per scheme per year, you may be able to self-assess by completing one of the PCI DSS Self-Assessment Questionnaire (SAQ); this is a self-assessment tool to assess security for cardholder data.
-
-When using GOV.UK Pay, the SAQ A questionnaire should apply. You should be eligible to complete the SAQ A questionnaire if you fulfil all the eligibility criteria and comply with SAQ A requirements 2, 8, 9 and 12:
-
-- the SAQ A questionnaire can be found in the [PCI documents library](https://www.pcisecuritystandards.org/document_library) [external link]
-- more detailed information on meeting requirements 2 and 8 can be found in [this article](https://pcissc.secure.force.com/faq/articles/Frequently_Asked_Question/How-do-PCI-DSS-Requirements-2-and-8-apply-to-SAQ-A-merchants) [external link]
-- more detailed information on the eligibility criteria can be found in the table below
+If you process fewer than 6 million transactions per scheme per year, you may
+be able to self-assess by completing one of the PCI DSS Self-Assessment
+Questionnaire (SAQ). This is a self-assessment tool to assess security for
+cardholder data.
+
+When using GOV.UK Pay, the SAQ A questionnaire should apply. You should be
```

Comment:
Best place to link to?

Reply:
for SAQ? https://www.pcisecuritystandards.org/pci_security/completing_self_assessment

Reply:
actually, if it's to find the SAQ questionnaire, it probably is best to stick with the document library link (https://www.pcisecuritystandards.org/document_library) because there's a header for all the SAQ questionnaires